dtcm/applesmash/map.xml

```diff
@@ -0,0 +1,3401 @@
+        </kit>
+    </action>
+    <trigger filter="pickup-red2-filter-2-by-green" action="pickup-red2-action-2-by-green" scope="player"/>
+    <action id="pickup-red2-action-3-by-green" scope="player">
+        <sound key="random.levelup" volume=".9" pitch="1.2"/>
+        <set var="pickup_red_momentary_variable" value="1"/>
+        <replace-item ignore-metadata="false" amount="1">
+            <find material="redstone block" name="Red Recovered block"/>
+            <replace material="air"/>
+        </replace-item>
+        <kit force="true" filter="always" deduct-items="false">
+            <item material="redstone block" name="Red Stolen block" amount="1"/>
+        </kit>
+    </action>
+    <trigger filter="pickup-red2-filter-3-by-green" action="pickup-red2-action-3-by-green" scope="player"/>
+    <!--
+         Pitch Guide:
+         0.5    F#      1      F#
+         0.53   G       1.059  G
+         0.561  G#      1.122  G#
+         0.595  A       1.189  A
+         0.63   A#      1.26   A#
+         0.667  B       1.335  B
+         0.707  C       1.414  C
+         0.749  C#      1.498  C#
+         0.794  D       1.587  D
+         0.841  D#      1.682  D#
+         0.891  E       1.782  E
+         0.944  F       1.888  F
+    -->
+    <!-- Four melodic sequences augment the game, using Hungarian/Japanese/Major style scales
+         - Game Intro - Theme-setting pentatonic with 4th harmonies for match
+         - Team Upgrade - Ascending rhythmic fanfare for team
+         - Picked up Their Monument Block - Ascending hungarian-minor fanfare for player
+         - Picked up My Monument Block - Descending hungarian-minor fanfare for player -->
+    <action id="red-pickup-red-action" scope="player">
+        <sound key="entity.firework.twinkle" volume="1" pitch="1.22"/>
+        <set var="pickup_me_variable" value="6"/>
+        <sound key="note.harp" volume=".5" pitch="1.122"/> <!-- g# (first note) -->
+        <message text="`e`l▲ `6Purchase `lteam upgrades`r`6 with Red blocks in the store!`r"/>
+    </action>
+    <trigger filter="red-pickup-red-filter" action="red-pickup-red-action" scope="player"/>
+    <action id="green-pickup-green-action" scope="player">
+        <sound key="entity.firework.twinkle" volume="1" pitch="1.22"/>
+        <sound key="note.harp" volume=".5" pitch="1.122"/> <!-- g# (first note) -->
+        <set var="pickup_me_variable" value="6"/>
+        <message text="`e`l▲ `6Purchase `lteam upgrades`r`6 with Green blocks in the store!`r"/>
+    </action>
+    <trigger filter="green-pickup-green-filter" action="green-pickup-green-action" scope="player"/>
+    <action id="red-pickup-green-action" scope="player">
+        <sound key="entity.firework.twinkle" volume="1" pitch="1.22"/>
+        <sound key="note.harp" volume=".5" pitch=".561"/> <!-- g# (first note) -->
+        <set var="pickup_them_variable" value="6"/>
+        <message text="`e`l▲ `6Take the Red block to store for `lplayer upgrades`r`6!`r"/>
```

Comment:
While I was testing the map, I got this message while I was on the Fuji team only when I broke the emerald apple monument. Are the messages reversed or is there a different red block that you could clarify more on?